test/object-store/util/test_file.cpp

```diff
@@ -70,7 +70,7 @@ TestFile::TestFile()
     disable_sync_to_disk();
     m_temp_dir = util::make_temp_dir();
     path = (fs::path(m_temp_dir) / "realm.XXXXXX").string();
-    util::Logger::set_default_level_threshold(realm::util::Logger::Level::TEST_LOGGING_LEVEL);
+    util::Logger::set_default_level_threshold(TestSyncManager::default_log_level());
     if (const char* crypt_key = test_util::crypt_key()) {
         encryption_key = std::vector<char>(crypt_key, crypt_key + 64);
     }
```

Comment:
Don't forget to address line 200 of this file, since it also uses the `TEST_ENABLE_LOGGING` variable, which is no longer being set.
```
#if TEST_ENABLE_LOGGING
                   m_logger = util::Logger::get_default_logger();
#else
                   // Logging is disabled, use a NullLogger to prevent printing anything
                   m_logger.reset(new util::NullLogger());
#endif
```